packages/styles/pagination.css

```
@@ -66,3 +66,13 @@
 .cauldron--theme-dark .Pagination [role='log'] {
   color: var(--accent-light);
 }
+
+@media (max-width: 450px) {
```

We tend to use rems:
```suggestion
@media (max-width: 28.125rem) {
```

Example: 
https://github.com/dequelabs/cauldron/blob/f7f6318a79b59471b15f518f3825a86604d06fe9/packages/styles/two-column-panel.css#L242